client/src/components/UserProfile.js

```diff
@@ -8,6 +8,8 @@ const UserProfile = (props) => {
   return (
     <div>
       <h1>Parkview profile for {userName}</h1>
+      <Link to="/parks/new">Add a new Park!</Link>
```

Comment:
This is linking to `/parks/new`, which I actually think is great, but it IS mismatching your current route :)